app/src/main/dist/config/crate.yml

```diff
@@ -860,3 +860,17 @@
 
 #psql.enabled: true
 #psql.port: 5432
+
+
+######################### Enterprise Features ########################
+
+#license.enterprise: true
+
+# To use any of the modules in this section, Crate.io must have given you
+# permission to enable and use such Enterprise Features (see
+# https://crate.io/enterprise/) and you must have a valid Enterprise or
+# Subscription Agreement with Crate.io. If you enable or use the Enterprise
+# Features, you represent and warrant that you have a valid Enterprise or
+# Subscription Agreement with Crate.io. Your use of the Enterprise Features if
```

Comment:
is this *if* right? this sentence doesn't make too much sense to me, but maybe it is just my lack of english.